src/reactapp/src/pages/Main.js

```diff
@@ -0,0 +1,152 @@
+import {
+    Table,
+    Thead,
+    Tbody,
+    Tfoot,
+    Tr,
+    Th,
+    Td,
+    TableCaption,
+    TableContainer,
+    Button,
+    Box,
+    Flex,
+    Stack,
+    Heading,
+    useColorModeValue,
+} from "@chakra-ui/react";
+import "../App.css";
+
+//Created a table to display the data from react objects
+export default function EvaluationTable() {
+    const data = [
+        {
+            evaluation: "T-TESS",
+            candidate: "Steven Mo",
+            evaluator: "Jane Doe",
+            date: new Date("2022-09-01"),
+            completed: true,
+        },
+        {
+            evaluation: "A-TESS",
+            candidate: "JOJO",
+            evaluator: "Jane Doe",
+            date: new Date("2021-09-02"),
+            completed: true,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+        {
+            evaluation: "B-TESS",
+            candidate: "Aidan Johnson",
+            evaluator: "Kevin Clear",
+            date: new Date("2023-09-03"),
+            completed: false,
+        },
+    ];
+
+    return (
+        <Stack spacing={8}>
+            <Stack align={"center"}>
+                <Heading mt="24px" fontSize={"5xl"}>
+                    Evaluations
+                </Heading>
+            </Stack>
+            <TableContainer style={{ maxWidth: "100%" }} className="responsiveTableContainer">
```

Comment:
100% maxwidth usually doesn't look very good - better to have a little bit of "white space" (even if it is actually another color) around the edges. Please try 95%, or add left and right margins. I would try `5rem` and see how that looks, then adjust up or down for a more please effect.